docs/installing-solidity.rst

```diff
@@ -9,11 +9,18 @@ Installing the Solidity Compiler
 Versioning
 ==========
 
-Solidity versions follow `semantic versioning <https://semver.org>`_ and in addition to
-releases, **nightly development builds** are also made available.  The nightly builds
-are not guaranteed to be working and despite best efforts they might contain undocumented
-and/or broken changes. We recommend using the latest release. Package installers below
-will use the latest release.
+Solidity versions mostly follow `Semantic Versioning <https://semver.org>`_.  We make
```

Comment:
Why `mostly`?

Reply:
Ah ok, because of the below. Maybe we should use something like "npm's flavour of semver"?

Reply:
The official notes from NPM on SemVer are at https://docs.npmjs.com/about-semantic-versioning

They no longer recommend doing 0.x releases.

Reply:
Addressed below